### DISCUSSION TITLE
Modelling SEI Growth on Particle Cracks Help

### DISCUSSION THREAD

**Discussion by robbiemacinnes** [pybamm-team/PyBaMM#3499] [Q&A]
Hi, I am fairly new to PyBaMM so I apologise if this is a simple question however, I have been going through several of the model examples such as modelling SEI Growth on Particle Cracks (https://docs.pybamm.org/en/stable/source/examples/notebooks/models/SEI-on-cracks.html) and I seem to be having an issue with the final output. As shown in the example notebook, the voltage figure comparing with cracking to without cracking represents a clear difference:

![Untitled](https://github.com/pybamm-team/PyBaMM/assets/121569686/c6fc96ed-5b7c-4812-8dbe-2b93a0b70c04)

However, regardless if I type the code line by line, download the notebook or run it all in colab, the results present the following:

![image](https://github.com/pybamm-team/PyBaMM/assets/121569686/b3e39197-2792-4f3e-a27a-705325d6792d)

One change that I do need to make to the code is changing the "Loss of lithium to negative SEI [mol]" and "Loss of lithium to negative SEI on cracks [mol]" to "Loss of lithium to SEI [mol]" and "Loss of lithium to SEI on cracks [mol]" for the code to run but I don't understand how this would affect my voltage output. 

Once again apologies if this has a simple answer but I would appreciate if someone could explain why this is happening.

**Comment by valentinsulzer** [comment]
@DrSOKane should be able to help with this

**Comment by DrSOKane** [comment]
@robbiemacinnes The reason why the voltage curves (left hand side of the figure) are so different is that if you have paticle mechanics enabled, the stress-assisted diffusion model is included by default. If you have no particle mechanics enabled, stress-assisted diffusion cannot be included, because the equation for the stress is not solved. The loss of lithium (right hand side of the figure) is behaving correctly. I didn't realize this when I wrote the notebook but it's clear to me now.
